test/go-tests/test_resourceservice.go

```diff
@@ -16,8 +17,8 @@ import (
 const configurationServiceBasePath = "/v1/project"
 
 func Test_ResourceServiceBasic(t *testing.T) {
-	projectName := "resource-service-test-project"
-	nonExistingProjectName := "non_existing_project"
+	projectName := osutils.GetOSEnv(KeptnNamespacePrefix) + "-resource-service-test-project"
```

Comment:
I would add something like the following to explain this line of code and add a default to don't have projects starting with `-` when tests are run locally
```suggestion
	// this is to avoid name aliasing with multiple CI parallel run of the integration tests
	newProjectName := osutils.GetOSEnvOrDefault(KeptnNamespacePrefix, DefaultKeptnNamespace) + "-resource-service-test-project"
```